eventuals/task.h

```diff
@@ -510,6 +595,13 @@ struct Task {
       template <typename... Args_>
       using With = TaskFromToWith<From_, To_, Args_...>;
 
+      To(
+          std::conditional_t<
+              std::is_void_v<To_>,
+              std::monostate,
+              To_> value)
```

Comment:
Ok, I'm seeing this one enough that I think we give it a name at the top-level:

```cpp
template <typename T>
using MonostateIfVoidOr = std::conditional_t< ...>;
```

Then:

```suggestion
      To(MonstateIfVoidOr<To_> value)
```